requirements/edx/development.in

```diff
@@ -21,6 +21,8 @@ snakefood                           # Lists dependencies between Python modules,
 sphinx==1.8.5                       # Pinned because 2.0.0 release requires Python '>=3.5' but current Python is 2.7.12
 vulture                             # Detects possible dead/unused code, used in scripts/find-dead-code.sh
 modernize                           # Used to make Python 2 code more modern with the intention of eventually porting it over to Python 3.
+#sphinxcontrib-openapi               # To generate API documentation
+git+https://github.com/edx/sphinxcontrib-openapi.git@nedbat/markdown-support#egg=sphinxcontrib-openapi[markdown]==0.0
```

Comment:
Please link to the upstream PR here so we can more easily check to see if we can switch back to upstream.

Reply:
done